replit_river/websocket_wrapper.py

```diff
@@ -5,6 +5,7 @@
 from websockets import WebSocketCommonProtocol
 
 logger = logging.getLogger(__name__)
+_background_tasks: set[asyncio.Task] = set()
```

Comment:
_fine_. don't love the idea of having another global, but the alternative is to re-add all the complexity. lgtm!